MapboxNavigation/MGLStyle.swift

```diff
@@ -25,59 +31,45 @@ extension MGLStyle {
      */
     public class var navigationNightStyleURL: URL {
         get {
-            if(MGLAccountManager.hasChinaBaseURL){
-                return mapboxChinaDayStyleURL
+            if MGLAccountManager.hasChinaBaseURL {
+                return mapboxChinaNightStyleURL
             }
-            return URL(string:"mapbox://styles/mapbox-map-design/ckd6dnz2q0m0q1io1mssumxqd")!
-        }
-    }
-    
-    /**
-     Returns the URL to the given version of the navigation guidance style. Available version are 1, 2, 3, and 4.
-     
-     We only have one version of navigation guidance style in China, so if you switch your endpoint to .cn, it will return the default day style.
-     */
-    public class func navigationGuidanceDayStyleURL(version: Int) -> URL {
-        if(MGLAccountManager.hasChinaBaseURL){
-            return mapboxChinaDayStyleURL
+            return defaultNavigationNightStyleURL
         }
-        return URL(string:"mapbox://styles/mapbox/navigation-guidance-day-v\(version)")!
     }
     
     /**
-     Returns the URL to the given version of the navigation guidance style. Available version are 2, 3, and 4.
+     Returns the URL to the given version of the Mapbox Navigation Day style. Available versions are 1, 2, 3, 4 and 5.
      
-     We only have one version of navigation guidance style in China, so if you switch your endpoint to .cn, it will return the default night style.
+     We only have one version of Mapbox Navigation Day style in China, so if you switch your endpoint to .cn, it will return the default day style.
      */
-    public class func navigationGuidanceNightStyleURL(version: Int) -> URL {
-        if(MGLAccountManager.hasChinaBaseURL){
-            return mapboxChinaNightStyleURL
-        }
-        return URL(string: "mapbox://styles/mapbox/navigation-guidance-night-v\(version)")!
-    }
-    
-    /**
-     Returns the URL to the given version of the Mapbox Navigation Preview Day style. Available versions are 1, 2, 3, and 4.
-     
-     We only have one version of Navigation Preview style in China, so if you switch your endpoint to .cn, it will return the default day style.
-     */
-    public class func navigationPreviewDayStyleURL(version: Int) -> URL {
+    public class func navigationDayStyleURL(version: Int) -> URL {
         if MGLAccountManager.hasChinaBaseURL {
             return mapboxChinaDayStyleURL
         }
-        return URL(string:"mapbox://styles/mapbox/navigation-guidance-day-v\(version)")!
+        
+        if (1...4).contains(version) {
```

Comment:
Cool. I think you can also use the `~=` operator here, but either way is more convenient than having to `&&` two conditions and risking an off-by-one error.